lib/cucumber/configuration.rb

```diff
@@ -189,7 +189,7 @@ def support_to_load
     def all_files_to_load
       files = require_dirs.map do |path|
         path = path.tr('\\', '/') # In case we're on windows. Globs don't work with backslashes.
-        path = path.gsub(/\/$/, '') # Strip trailing slash. # rubocop:disable Style/RegexpLiteral
+        path = path.gsub(%r{/$}, '') # Strip trailing slash.
```

Comment:
We are allowed to use `/` inside the standard `//` regex matchers if we escape it. i.e. `/\//`

Reply:
Yes. But that is responsible for a rubocop offense.
Per default, Style/RegexpLiteral enforces using slashes, but do not allow inner slashes thus requires using %r with inner slashes. Cf. https://docs.rubocop.org/rubocop/1.0/cops_style.html#styleregexpliteral

I've let the default rubocop configuration here, but fix the ofenses without having magic comments.

If we prefer avoiding the use of %r, we may consider configuring the cop by allowing inner slashes.